CMakeLists.txt

```diff
@@ -71,10 +90,7 @@ target_include_directories(rmm INTERFACE "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOUR
 
 if(CUDA_STATIC_RUNTIME)
   message(STATUS "RMM: Enabling static linking of cudart")
-  target_link_libraries(rmm INTERFACE CUDA::cudart_static)
```

Comment:
@robertmaynard Is this change to rely on `rapids_cuda_init_runtime` valid, or is it not valid for RMM because it doesn't have any source files (header only)?

Reply:
It isn't valid for rmm since it is header only.